packages/firebase_auth/firebase_auth/pubspec.yaml

```
@@ -36,6 +37,12 @@ flutter:
       ios:
         pluginClass: FLTFirebaseAuthPlugin
       macos:
-        pluginClass: FLTFirebaseAuthPlugin
+        # pluginClass: FLTFirebaseFunctionsPlugin
+        # Temporary for development of Desktop packages
+        default_package: firebase_core_desktop
+      linux:
+        default_package: firebase_core_desktop
+      windows:
+        default_package: firebase_core_desktop
```

I think these are incorrect maybe, should they be this instead:

```suggestion
        default_package: firebase_auth_desktop
      linux:
        default_package: firebase_auth_desktop
      windows:
        default_package: firebase_auth_desktop
```

---

actually they have to be `firebase_auth_desktop`